src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -167,11 +175,22 @@ public async Task<IActionResult> Index(TenantIndexOptions options, PagerParamete
             };
 
             // We populate the SelectLists
+            model.Options.TenantsCategories = allSettings
+                .GroupBy(t => t["Category"])
+                .Where(t => !String.IsNullOrEmpty(t.Key))
+                .Select(t => new SelectListItem(t.Key, t.Key))
+                .ToList();
+
+            model.Options.TenantsCategories.Insert(0, new SelectListItem(
+                S["All"],
+                DefaultCategory,
```

Comment:
```suggestion
                String.Empty,
```

And remove `DefaultCategory` - it works, and we do this everywhere else.

Reply:
Same thing